selendroid-server/src/main/java/io/selendroid/server/model/AndroidNativeElement.java

```diff
@@ -254,18 +256,73 @@ public String getText() {
   public void click() {
     waitUntilIsDisplayed();
     scrollIntoScreenIfNeeded();
+
+    final View view = getView();
+
+    final int[] xy = new int[2];
+
+    if (Build.VERSION.SDK_INT < 19 || view.isLaidOut()) {
```

Comment:
I actually didn't know about that call. That seems like a great way to reduce the flakiness of clicks.